RNSound/RNSound.m

```diff
@@ -21,17 +21,17 @@ - (void)audioSessionChangeObserver:(NSNotification *)notification {
         [userInfo[@"AVAudioSessionInterruptionTypeKey"] longValue];
     AVAudioPlayer *player = [self playerForKey:self._key];
     if (audioSessionInterruptionType == AVAudioSessionInterruptionTypeEnded) {
-        if (player && player.isPlaying) {
+        if (player) {
```

Comment:
Remove the `isPlaying` check because the player is paused when the begin event is received.